### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by cristea2017** [MissThee/threejs-minecraft#6]
This is amazing. 
It's possible to import minecraft world intro three js-minecraft ?

**Comment by MissThee** [comment]
Sorry, there is no such function, but it is possible to some extent.

Why can't it?
1. This demo now only has a small part of the blocks in Minecraft, and they just look similar in appearance, without any other properties of the blocks, for example, the water blocks cannot flow. (Because I can't think of the development direction of this demo, and adding properties to each box is a lot of work, so I stopped writing)
2. There are still some performance issues that have not been resolved. When the number of squares reaches several hundred, significant frame drop will occur. (At first I made this demo because I was attracted by some works of threejs, so I did this while learning, so I don’t know how to optimize it further)

Why is it possible?
There are also some not so friendly messages that can help you realize the export function. If the following three conditions can be fulfilled, some scenes in MC can be restored:
1. Add missing blocks to this demo
2. Export only a small piece of scene
3. Export the scene data from minecraft and organize it into the specified json format, like [[x, y, z, "grass"], [x, y, z, "stone"], ....].
Therefore, this method is very troublesome and it is not recommended to try it. . .
